lerobot/common/datasets/lerobot_dataset.py

```diff
@@ -144,7 +144,7 @@ def __getitem__(self, idx):
                 self.tolerance_s,
             )
 
-        if self.video:
+        if self.video and self.video_backend != "raw":
```

Comment:
We should return directly otherwise code's gonna break right after on the image transforms as it will expect those frames to be in the `item` dict.

Nit: I suggest calling this backend `rerun` to be more explicit.
```suggestion
        if self.video_backend == "rerun":
            return item

        if self.video:
```